marketplaceItems/avatarCustomization101/appResources/appData/resources/modules/mirrorClient.js

```diff
@@ -24,9 +25,9 @@
     var MAX_MIRROR_RESOLUTION_SIDE_PX = 1500;        // The max pixel resolution of the long side of the mirror
     var ZERO_ROT = { w: 1, x: 0, y: 0, z: 0 };   // Constant quaternion for a rotation of 0
     var FAR_CLIP_DISTANCE = 5;     // The far clip distance for the spectator camera when the mirror is on
-    var mirrorOverlayID;            // The entity ID of the overlay that displays the mirror reflection
-    var mirrorOverlayRunning;       // True if mirror overlay is reflecting, false otherwise
-    var mirrorOverlayOffset = 0.01; // The distance between the center of the mirror and the mirror overlay
+    var mirrorLocalEntityID = false;            // The entity ID of the local entity that displays the mirror reflection
+    var mirrorLocalEntityRunning;       // True if mirror local entity is reflecting, false otherwise
+    var mirrorLocalEntityOffset = 0.005; // The distance between the center of the mirror and the mirror local entity
```

Comment:
Are you sure that this isn't too close of an offset? I think it'll be too close if users are standing a little further away than "normal" from the mirror.

Reply:
i can put it back